templates/kapeta/block-type-cli/rest-clients-index.ts.hbs

```diff
@@ -0,0 +1,22 @@
+{{#consumes 'kapeta/resource-type-rest-client'}}
+    //#FILENAME:src/.generated/clients/index.ts:write-always
+    import hostsConfig from '../../../config/hosts.development.json';
```

Comment:
No way to use a "production" version?

Reply:
In desktop we set NODE_END to development in npm run start:dev - and if that value is not set or not development - we use production

Reply:
Good point, forgot about the "production" part. Will fix this!